Jasmine/Common/ViewControllers/HomeScreenViewController.swift

```diff
@@ -3,7 +3,13 @@ import UIKit
 class HomeScreenViewController: UIViewController {
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
         if let gridGame = segue.destination as? GridGameViewController {
-            gridGame.segueWith(GridViewModel())
+            guard let gameDataFactory = try? GameDataFactory() else {
```

Comment:
Just adding a notice here for @xdrawks and @riwu. These should and will be moved later. Until then, leaving it here is fine.